nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/PutHBaseRecord.java

```diff
@@ -207,6 +211,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             final String details = String.format("Put %d cells to HBase.", columns);
             session.getProvenanceReporter().send(flowFile, urls.get(0), details, sendMillis);
             session.getProvenanceReporter().send(flowFile, urls.get(1), details, sendMillis);
```

Comment:
We still have two calls to send a provenance event, I think we can get rid of the second one, and then we still need to send an event before transferring to failure.